docs/publish-module/publish-information/resign-binary.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Resign Binary
+description: Learn how to resign your iOS application binaries within Appcircle to change provisioning profiles or app entitlements.
+tags: [iOS, resigning, provisioning profiles, entitlements]
+---
+
+import Screenshot from '@site/src/components/Screenshot';
+
+# Resign Binary
+
+The **Resign Binary** feature in Appcircle allows you to resign both iOS and Android application binaries. For iOS applications, you can use new provisioning profiles or modify the app's entitlements, which is useful for adjusting the app’s capabilities or updating its distribution settings without requiring a new build. For Android applications, you can resign your binaries with a new keystore, allowing you to update the app's signing credentials crucial for app distribution and updates.
+
+This feature streamlines the process of updating app distribution and security settings, ensuring that your applications can be quickly adapted to meet changing requirements or distribution strategies.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/be-3161-publish-resign-option.png' />
+
+## Resign iOS Binary
+
+When you need to distribute an iOS application to different environments (like QA, staging, or production) or need to change the app’s entitlements, the **Resign Binary** feature simplifies this process. You can resign an app binary with a new provisioning profile that matches the intended distribution certificate.
+
+### Fields and Options
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/be-3161-publish-resign.png' />
+
+#### Display Name
+
+- **Field**: `CFBundleDisplayName`
+- **Description**: The user-visible name for the bundle, used by Siri and visible on the iOS Home screen.
+
+#### Version
+
+- **Field**: `CFBundleShortVersionString`
+- **Description**: The release or version number of the bundle.
+
+#### Build Number
+
+- **Field**: `CFBundleVersion`
+- **Description**: The version of the build that identifies an iteration of the bundle.
+
+#### Entitlements
+
+- **Options**:
+  - **Entitlements from provisioning profiles**: Use entitlements from the new provisioning profile.
+  - **Combine app entitlements**: Extract app bundle code signing entitlements and combine with entitlements from new provisioning profiles.
+  - **New app entitlements**: Edit this XML file to edit capabilities. Resigning process uses a single entitlement XML file.
+
+#### Targets
+
+- **Description**: Select a new provisioning profile for new targets.
+
+#### Bundle ID
+
+- **Field**: `CFBundleIdentifier`
+- **Description**: Original Bundle ID.
+
```

Comment:
:::caution BundleID

Please note that **BundleID** changes are not allowed while the related version is resigned in the **Publish module**. If you want to change the **BundleID value** of your package, please use the **Resign Binary** feature in [**Testing Distribution**](/distribute/platform-specific-guidance/ios/resigning-ios-binaries).

:::